src/instance/minecraft/chat/QuestChat.ts

```diff
@@ -0,0 +1,25 @@
+import { MinecraftChatContext, MinecraftChatMessage } from '../common/ChatInterface'
+import { LOCATION, SCOPE } from '../../../common/ClientInstance'
+import { ColorScheme } from '../../discord/common/DiscordConfig'
+import { EventType } from '../../../common/ApplicationEvent'
+
+export default {
+  onChat: function (context: MinecraftChatContext): void {
+    const regex = /^GUILD QUEST TIER [1-9] COMPLETED!/g
+
+    const match = regex.exec(context.message)
```

Comment:
Trim required since message sometimes has 18 space before the display text.
```suggestion
    const match = regex.exec(context.message.trim())
```